pymatgen/io/lobster/lobsterenv.py

```diff
@@ -227,6 +227,17 @@ def get_anion_types(self):
 
         return set(anion_species)
 
+    def get_anion_types(self):
+        """
+        Return the types of anions present in crystal structure as a set
+        Returns: set of Element describing anions in the crystal structure
+        """
+        from warnings import warn
+
+        warn("get_anion_types will be deprecated. Please use anion_types instead.")
```

Comment:
monty has a [decorator `deprecated`](https://pythonhosted.org/monty/monty.html#monty.dev.deprecated) for this:

```suggestion
        @deprecated(replacement=self.anion_types, message="use anion_types instead")
        def get_anion_types(self):
```